src/lib.rs

```diff
@@ -5,13 +5,15 @@
 #![deny(clippy::unused_async)]
 #![deny(clippy::unnecessary_wraps)]
 
-use std::io::ErrorKind;
 use std::path::Component;
 use std::path::Path;
 use std::path::PathBuf;
+use sys_traits::SystemRandom;
 use thiserror::Error;
 use url::Url;
 
+pub mod fs;
```

Comment:
This should arguably be in a separate crate, but I don't think it's worth the additional maintenance burden.

Reply:
I agree it's not worth it.